contrib/depends/packages/darwin_sdk.mk

```diff
@@ -1,8 +1,8 @@
 package=darwin_sdk
-$(package)_version=11.1
-$(package)_download_path=https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/
-$(package)_file_name=MacOSX$($(package)_version).sdk.tar.xz
-$(package)_sha256_hash=68797baaacb52f56f713400de306a58a7ca00b05c3dc6d58f0a8283bcac721f8
+$(package)_version=12.2
+$(package)_download_path=https://featherwallet.org/files/SDK/
```

Comment:
I would prefer if we could set this to

https://bitcoincore.org/depends-sources/sdks

instead, I get like 10x the speeds there.